NUnitConsole.sln

```diff
@@ -96,6 +97,18 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "deprecated", "deprecated",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "nunit.engine.core", "src\NUnitEngine\nunit.engine.core\nunit.engine.core.csproj", "{A19C026B-1C0F-4AA3-AC49-7D8B4C7231CF}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build-scripts", "build-scripts", "{2DA0963D-BD5E-41A6-8B27-FA312E1254C4}"
+	ProjectSection(SolutionItems) = preProject
+		build-scripts\build-parameters.cake = build-scripts\build-parameters.cake
+		build-scripts\constants.cake = build-scripts\constants.cake
+		build-scripts\package-checks.cake = build-scripts\package-checks.cake
+		build-scripts\package-tests.cake = build-scripts\package-tests.cake
+		build-scripts\test-results.cake = build-scripts\test-results.cake
```

Comment:
```suggestion
		build-scripts\test-results.cake = build-scripts\test-results.cake
		build-scripts\testing.cake = build-scripts\testing.cake
```

Reply:
@mikkelbu I don't see what the change is here.

Reply:
I also included the line `build-scripts\testing.cake = build-scripts\testing.cake`